level_file_templates/details.yaml

```diff
@@ -0,0 +1,57 @@
+'1':
+  message: "Initial commit"
+'2':
+  message:
+    - Start solving an issue on a branch
+    - Solves issue 1
+    - Solves issue 2
+'3':
+  message:
+    - This poriton of the message will be two lines below the first line
+    - This line is below the line above
+  add-files:
+    file.txt:
+      - "'add-files' will specify file names and add content"
+      - These lines will show up in file.txt
+    another.txt:
+      - "'add-files' and 'remove-files' are meant to be used together"
+  remove-files:
```

Comment:
I'm not fully clear on what remove-files is doing in this case. Is it removing all files that were added in commits '1' and '2'?